### ISSUE TITLE
Provide release with fix for issue 506

### ISSUE THREAD

**Issue by moritz89** [jenkinsci/gitlab-plugin#618]
## Issue
Create a new release including merge https://github.com/jenkinsci/gitlab-plugin/pull/548 which is issue https://github.com/jenkinsci/gitlab-plugin/issues/506

### Context
- **Gitlab plugin version**: 1.4.8
- **Gitlab version**:  8.15.3
- **Jenkins version**: 2.40
- **Job type**: Pipeline

### Problem description

Would like to request that a new release with this fix is performed. According to the readme, it should just be `mvn release:prepare release:perform`

cc @omehegan

**Comment by omehegan** [comment]
Thanks for bringing this to my attention, that PR was merged into the wrong branch. #622 will resolve this when it is merged. Blocked on #614 right now.

**Comment by omehegan** [comment]
Please watch #622 for updates about this issue.
